build.sbt

```diff
@@ -19,10 +19,15 @@ lazy val root = project.in(file("."))
 
 lazy val `scala-parser-combinators` = crossProject.in(file(".")).
   settings(scalaModuleSettings: _*).
+  jvmSettings(scalaModuleSettingsJVM).
   settings(
-    name := "scala-parser-combinators-root",
+    repoName := "scala-parser-combinators",
```

Comment:
It doesn't cause harm because the `name` is picked up from the project name, so you get

```
> name
[info] scala-parser-combinatorsJS/*:name
[info] 	scala-parser-combinators
[info] scala-parser-combinatorsJVM/*:name
[info] 	scala-parser-combinators
```
But we should fix it anyway.